lib/wallet/open_channel.dart

```diff
@@ -25,131 +24,148 @@ class OpenChannel extends StatefulWidget {
+                    Padding(
+                      padding: EdgeInsets.only(left: 20.0, right: 20.0),
+                      child: Divider(),
+                    )
+                  ]),
+                  const SizedBox(height: 10),
+                  const Center(child: Icon(Icons.private_connectivity_outlined, size: 80)),
+                  const SizedBox(height: 10),
+                  ListTile(
+                      leading: addCircle("1"),
+                      title: const Text("Fund your bitcoin wallet"),
+                      subtitle: const Text(
+                          "In order to establish a Lightning channel between 10101 and you we need to lock some bitcoin on chain.",
+                          style: TextStyle(color: Colors.grey))),
+                  const SizedBox(height: 15),
+                  ListTile(
+                      leading: addCircle("2"),
+                      title: const Text("Enter the channel amount"),
+                      subtitle: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Text(
+                              "Define the amount of bitcoin you want to lock into the Lightning channel. 10101 will double it for great inbound liquidity. The max amount you can use is currently $maxTakerChannelAmount sats.",
+                              style: const TextStyle(color: Colors.grey)),
+                          TextFormField(
+                            initialValue: ThousandsSeparatorInputFormatter()
+                                .formatEditUpdate(TextEditingValue.empty,
+                                    TextEditingValue(text: maxTakerChannelAmount.toString()))
+                                .text,
+                            readOnly: submitting,
+                            keyboardType: TextInputType.number,
+                            decoration: const InputDecoration(
+                              border: UnderlineInputBorder(),
+                            ),
+                            inputFormatters: <TextInputFormatter>[
+                              FilteringTextInputFormatter.digitsOnly,
+                              ThousandsSeparatorInputFormatter(),
+                            ],
+                            onChanged: (value) {
+                              setState(() {
+                                try {
+                                  takerChannelAmount = int.parse(value.replaceAll(",", ""));
+                                } on Exception {
+                                  takerChannelAmount = 0;
+                                }
+                              });
+
+                              _formKey.currentState!.validate();
+                            },
+                            validator: (value) {
+                              if (value == null) {
+                                return "Enter channel amount";
```

Comment:
🤡 I find `channel amount` (or channel size) a bit vague. What the user is inputting is their outbound liquidity. Perhaps `Enter channel outbound liquidity`?

Reply:
Same issue with other usages of `amount` in this file.

Reply:
I changed it to `outbound liquidity` in most places.